api/packages.ini

```diff
@@ -2,10 +2,8 @@
 [python]
 python3 = buster, bullseye, bionic, focal
 python3-pip = buster, bullseye, bionic, focal
-
-[django]
-uwsgi = buster, bullseye, bionic, focal
-uwsgi-plugin-python3 = buster, bullseye, bionic, focal
+gunicorn = buster, bullseye, bionic, focal
+python3-gunicorn = buster, bullseye, bionic, focal
```

Comment:
I wonder if we prefer to install/pin a version from pip for now and let the package maintainer install those packages.

Reply:
I'd rather use distro versions of packages where possible